README.md

```diff
@@ -1,25 +1,28 @@
 # monk
 
 [![build status](https://secure.travis-ci.org/Automattic/monk.png?branch=master)](https://secure.travis-ci.org/Automattic/monk)
+[![codecov](https://codecov.io/gh/Automattic/monk/branch/master/graph/badge.svg)](https://codecov.io/gh/Automattic/monk)
 
 Monk is a tiny layer that provides simple yet substantial usability
 improvements for MongoDB usage within Node.JS.
 
+*note*: monk 2.x drop the support for node < 0.12. If you are still using an earlier version, stick to monk 1.x
```

Comment:
I just tested on node 0.10 and it seems to work. But it might be that ava is putting a Promise polyfill or something, it shouldn't work.
And on node 0.8, 0.6, etc., it's not even possible to do `npm i`, the versioning in package.json is broken